convert2rhel/utils/__init__.py

```diff
@@ -868,7 +868,7 @@ def find_keyid(keyfile):
 
     :arg keyfile: The filename that contains the gpg key.
 
-    .. note:: rpm doesn't use the full gpg fingerprint so don't use that even though it would be
+    .. note:: rpm doesn't use the full gpg key_id so don't use that even though it would be
         more secure.
```

Comment:
```suggestion
    .. note:: rpm doesn't use the full gpg fingerprint so don't use that even though it would be
        more secure, instead use the key id.
```